### ISSUE TITLE
Report error at GitHub Pages site - how?

### ISSUE THREAD

**Issue by HebaruSan** [Kerbalism/Kerbalism#704]
A user on the forums is seeing this message:

![screenshot](https://imgur.com/dXMJO7Y.png)

https://forum.kerbalspaceprogram.com/index.php?/topic/198681-why-i-cant-make-kerbalism-work/

https://github.com/Kerbalism/Kerbalism/blob/df311ab026cc67bdd1b8e25652b775c15e225840/src/Kerbalism/System/Kerbalism.cs#L243

I visited https://kerbalism.github.io/ and I did not see a way to submit an issue there. It's just a standard static GitHub Pages site.

Should this say https://github.com/Kerbalism/Kerbalism/issues/new instead?

**Comment by gotmachine** [comment]
Yeah we should probably not mention github anyway. discord or the KSP forums are the primary places to go to get support.
